storage/src/mmr/mem.rs

```diff
@@ -260,8 +261,8 @@ mod tests {
             0,
             "oldest_required_element should return 0 on empty MMR"
         );
-
-        let element = Digest::from_static(b"01234567012345670123456701234567");
+        let element = Bytes::from_static(b"01234567012345670123456701234567");
```

Comment:
It would be great if we didn't have to use `Bytes` here? It seems like we should be able to do `try_from()` over the static [u8]?

Reply:
You are right, I did the changes 👍